buildSrc/src/test/resources/LICENSEHEADER

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
why do we need this? is the license header not already in `src/main/resources`?

Reply:
deleted via https://github.com/micronaut-projects/micronaut-guides/pull/1523/commits/dd96352f0f6a79069b13522fe42b5cfaa3f9d033